ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/JdbcLedgerDaoPostCommitValidationSpec.scala

```diff
@@ -34,22 +40,40 @@ private[dao] trait JdbcLedgerDaoPostCommitValidationSpec extends LoneElement {
       loggingContext: LoggingContext
   ): ResourceOwner[LedgerDao] = {
     val metrics = new Metrics(new MetricRegistry)
-    JdbcLedgerDao
-      .validatingWriteOwner(
+    val dbType = DbType.jdbcType(jdbcUrl)
+    val storageBackendFactory = StorageBackendFactory.of(dbType)
+    val participantId = Ref.ParticipantId.assertFromString("JdbcLedgerDaoPostCommitValidationSpec")
+    DbDispatcher
+      .owner(
+        dataSource = storageBackendFactory.createDataSourceStorageBackend.createDataSource(jdbcUrl),
         serverRole = ServerRole.Testing(getClass),
-        jdbcUrl = jdbcUrl,
-        connectionPoolSize = 16,
+        connectionPoolSize = dbType.maxSupportedWriteConnections(16),
         connectionTimeout = 250.millis,
-        eventsPageSize = eventsPageSize,
-        eventsProcessingParallelism = eventsProcessingParallelism,
-        servicesExecutionContext = executionContext,
         metrics = metrics,
-        lfValueTranslationCache = LfValueTranslationCache.Cache.none,
-        enricher = None,
-        participantId = Ref.ParticipantId.assertFromString("JdbcLedgerDaoPostCommitValidationSpec"),
-        compressionStrategy = CompressionStrategy.none(metrics),
-        ledgerEndCache = ledgerEndCache,
-        errorFactories = errorFactories,
+      )
```

Comment:
This configuration is repeated in multiple files. Can we extract it to a helper object for tests so that we wouldn't have to specify `connectionPoolSize` and `connectionTimeout` every time?

Reply:
I assume you are referring to the test-usages: I counted 4, with slight deviations. It is somehow forced to me to couple those tests together on some defaults.